tests/contrib/suitespec.yml

```diff
@@ -263,6 +263,11 @@ suites:
       - '@botocore'
       - tests/contrib/aiobotocore/*
     runner: riot
+    services:
+      - moto
+    env:
+      TEST_MOTO_PORT: '3000'
```

Comment:
```suggestion
```

only if the other suggestion is taken, only doing this to make it easier _if_ you wanted to take the changes.